contrib/postgresql/README.md

```diff
@@ -34,16 +34,24 @@ collectd/postgresql calles the collectd_insert function.
 			double precision[])		-- values
 ```
 
-This seems to represents the user_data_t/notification_t structure.
+This seems to represent the user_data_t/notification_t structure.
 https://github.com/collectd/collectd/blob/ef1e157de1a4f2cff10f6f902002066d0998232c/src/daemon/plugin.h#L172
 
-Lets take the ping plugin as an example. It collects 3 values: ping, ping_stddev, ping_droprate.
+Let's take the ping plugin as an example. It collects 3 values: ping, ping_stddev, ping_droprate.
 
-The current structure creates 3 identifiers and 3 lines for each entry. The identifiers get reused. It reports "192.168.myping.ip" as type.
+The current structure creates 3 identifiers and 3 lines for each entry. The identifiers get reused. It reports "192.168.200.123" as type.
 
-To draw a diagram with e.g. grafana i would like all 3 values near each other for that host that i am pinging. See the graph in the wiki. The current setup must join through all collected values to scrap the ping values out of it. Each value must do the same again because it has an other identifier.
+To draw a diagram with e.g. grafana I would like all 3 values near each other for that host that i am pinging. See the graph in the wiki. The current setup must join through all collected values to scrap the ping values out of it. Each value must do the same again because it has another identifier.
```

Comment:
```suggestion
To draw a diagram with e.g. grafana I would like all 3 values near each other for that host that I am pinging. See the graph in the wiki. The current setup must join through all collected values to scrap the ping values out of it. Each value must do the same again because it has another identifier.
```